### PR TITLE
update template PR

### PR DESCRIPTION
**Pull request by Mralexandresys**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

#### Description of Change
change url in template request
Add row check list for update package version

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -7,8 +7,9 @@
 
 Please ensure you've completed all of the following.
 
-- [ ] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
-- [ ] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
+- [ ] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
```
Comment (vraravam) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: Please undo the 1st two lines of changes. Though the issues get reported in the main (ferdium-app) repo, the contriuting guidelines and scripts to run are different between the 2 repos.

**Review by vraravam — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Mralexandresys) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: I understand what you mean but the url on the 1st line does not work. I used the CONTRIBUTING.md doc to install repo recipes on my local. 
Should we create a new CONTRIBUTING.md ?

Comment (mcmxcdev) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: @Mralexandresys is right, the link to https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md is broken.

Maybe I fired too quicky in merging https://github.com/ferdium/ferdium-recipes/pull/471, @vraravam let us know if something has to be updated.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -7,8 +7,9 @@
...
+- [ ] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
+- [ ] I update the version package [Updating](https://github.com/ferdium/ferdium-recipes/blob/main/docs/updating.md#4-updating-the-version-number)
```
Comment (mcmxcdev) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: Ah this is cool, then more ppl will hopefully do the version update right away without having to get reminded.

Comment (shiruken1) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Yes! I would've appreciated knowing about this prior to the commit hook yelling at me about it

**Review by SpecialAro — APPROVED** [review]

**Comment by SpecialAro** [comment]
@allcontributors - please add @Mralexandresys for infra

**Pull request merged**